test/byContract/ENSIntegration/ENSIntegration.test.ts

```diff
@@ -0,0 +1,126 @@
+// SPDX-FileCopyrightText: 2020-2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { deployments, ethers, expect } from 'hardhat';
+import { loadHardhatFixtureDeployment } from '../../../scripts/hardhat-fixture-deployer';
+import {
+    ENSRegistry,
+    ENSRegistry__factory,
+    IexecInterfaceNative,
+    IexecInterfaceNative__factory,
+    PublicResolver__factory,
+    ReverseRegistrar__factory,
+} from '../../../typechain';
+import { getIexecAccounts } from '../../../utils/poco-tools';
+const CONFIG = require('../../../config/config.json');
+
+describe('ENSIntegration', () => {
+    let proxyAddress: string;
+    let [iexecPoco, iexecPocoAsAdmin]: IexecInterfaceNative[] = [];
+    let [iexecAdmin, anyone]: SignerWithAddress[] = [];
+    let ensRegistry: ENSRegistry;
+
+    beforeEach('Deploy', async () => {
+        // Deploy all contracts
+        proxyAddress = await loadHardhatFixtureDeployment();
+        // Initialize test environment
+        await loadFixture(initFixture);
+    });
+
+    async function initFixture() {
+        ({ iexecAdmin, anyone } = await getIexecAccounts());
+        const ensRegistryAddress = (await deployments.get('ENSRegistry')).address;
+        ensRegistry = ENSRegistry__factory.connect(ensRegistryAddress, anyone);
+        iexecPoco = IexecInterfaceNative__factory.connect(proxyAddress, anyone);
+        iexecPocoAsAdmin = iexecPoco.connect(iexecAdmin);
+    }
+
+    describe('Forward resolution', () => {
+        it('Should resolve initial names', async () => {
```

Comment:
Is it possible to create a should not resolve test ? like pointing to address(0) if it's the default resolve address